### ISSUE TITLE
Update bitly API usage from the deprecated v3 to v4

### ISSUE THREAD

**Issue by edwinmp** [devinit/data-hub-spotlights#288]
Simon notified us about this on [Slack](https://devinitorg.slack.com/archives/CCG8Y0YJG/p1673604329679809). Support for v3 ends on January 31.

**Resources (suggestion - feel free to find your own)**

- https://dev.bitly.com/

**Comment by edwinmp** [comment]
This has been stalled by dependency updates that are required for it's deploying ... will make time for it.

**Comment by ThatcherK** [comment]
@edwinmp I remember working on this and finishing it..

**Comment by edwinmp** [comment]
Yeah ... failed to deploy. Current problem is unrelated to the fix. Its quite annoying really. Don't want to close this till it's deployed.
